### PR TITLE
Add schedule `Chrome` weekly check

### PR DESCRIPTION
**Pull request by logist322**
## Synopsis

This PR adds a new schedule workflow for checking the last stable version of the Chrome and creates a new automate PR with updated `version` and `commit`.

## Checklist

- Created PR:
    - [x] In [draft mode][l:1]
    - [x] Name contains `Draft: ` prefix
    - [x] Name contains issue reference
    - [x] Has `k::` labels applied
    - [x] Has assignee
- [x] Documentation is updated (if required)
- [x] Tests are updated (if required)
- [x] Changes conform code style
- [x] CHANGELOG entry is added (if required)
- [x] FCM (final commit message) is posted
    - [x] and approved
- [x] [Review][l:2] is completed and changes are approved
- Before merge:
    - [x] Milestone is set
    - [x] PR's name and description are correct and up-to-date
    - [x] `Draft: ` prefix is removed
    - [x] All temporary labels are removed

[l:1]: https://help.github.com/en/articles/about-pull-requests#draft-pull-requests
[l:2]: https://help.github.com/en/articles/reviewing-changes-in-pull-requests

### CODE REVIEW

**Comment by logist322** [comment]
## FCM
```
Add schedule `Chrome` weekly check (#5)
```

**Review by alexlapa — APPROVED** [review]

File: .github/workflows/chrome-check.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          {
+            read line;
+            sed -i "${line}s|.*|WEBRTC_COMMIT=${WEBRTC_HASH}|g" ./VERSION;
+          }
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GH_INSTRUMENTISTO_BOT_PAT }}
+          commit-message: Upgrade to ${{ env.LAST_VERSION }} version
```
Comment (tyranron) [inline .github/workflows/chrome-check.yml:75]: @logist322 do we have idempotency here? If the branch already exists, will this step succeed, or fail, or create a new PR?

File: .github/workflows/chrome-check.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                core.exportVariable('LAST_VERSION', commit.version);
+                core.exportVariable('WEBRTC_HASH', commit.hashes.webrtc);
+              });
+
+      - name: Check if the versions are different
+        run: |
+          if [ $LAST_VERSION = $CURRENT_VERSION ]
+          then
+            echo "The latest version ($CURRENT_VERSION) is already released!"
+            exit 1
```
[the remark this answers is not in the document]
Comment (logist322) [reply .github/workflows/chrome-check.yml]: @tyranron got it!

File: .github/workflows/chrome-check.yml (hunk 1 above)
Comment (tyranron) [reply .github/workflows/chrome-check.yml]: @logist322 no, I think the default behavior is idempotent enough.

**Comment by logist322** [comment]
@alexlapa review ping.

**Review by tyranron — CHANGES_REQUESTED** [review]

File: upgrade-ver.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+jq -r '"version:\(.[0].version)\nwebrtc:\(.[0].hashes.webrtc)"' |
+while read data; do
+    name=$(echo "$data" | cut -d':' -f1)
+    value=$(echo "$data" | cut -d':' -f2)
+    field=$([[ $name = version ]] && echo "WEBRTC_VERSION" || echo "WEBRTC_COMMIT")
+    grep -n $field ./VERSION |
+    cut -d':' -f1 |
+    {
+        read line
+        sed -i "${line}s|.*|${field}=${value}|g" ./VERSION
```
Comment (tyranron) [inline upgrade-ver.sh]: @logist322 implementation is unnecessarily complicated.

**Review by tyranron — APPROVED** [review]

**Pull request merged**